codecov.yml

```
@@ -0,0 +1,20 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 0%
```

Please set the threshold to a value of 85%

```suggestion
        threshold: 85%
```